docs/source/_static/parameter-files/parameter_CFIMP.prm

```diff
@@ -191,20 +189,20 @@ DOY_RANGE = 1 365
 # the window breakpoints. If you specify 5 breakpoints, there will be four
 # windows. The windows can extend to the previous/next year (e.g. 270 30
 # 91 181 270 would extend into the previous year, 1 30 91 181 270 30 would
-# extend into the next year.
+# extend into the next year).
 # Type: Integer list. Valid values: [1,365]
 SEASONAL_WINDOW = 1 91 181 271 365
 # This parameter defines the radius of the prediction kernel (in projection
 # units, commonly in meters). A larger kernel increases the chance of finding
 # a larger number of within-class pixels, but increases prediction time
-# Type: Double. Valid values: ]0,BLOCK_SIZE]
+# Type: Double. Valid values: [0,BLOCK_SIZE]
```

Comment:
same here. ]0 is intended

Reply:
I think the most intuitive way is to have a representation of valid values - as the text says.
At the moment it's: 'valid values' and is followed by an invalid value.
My eye didn't even pick up on the possibility that this could be a mathematical representation ;)